Flutter SDK is now in Alpha (#1342)
mongodb/docs-realm: source/sdk/flutter.txt

.. _flutter_limitations:

Alpha Limitations
-----------------

> Because this is an alpha version of the SDK, functionality is limited and there 

Reviewer: ```suggestion
Because this is an alpha version of the SDK, functionality is limited, and there 
```

are specific configuration considerations:

- The alpha version of the SDK requires a custom engine based on Flutter 2.0. 

>   For more information, and a diff of the changes, see the 

Reviewer: ```suggestion
  For a :wikipedia:`diff </Diff>` of the changes and additional information, see the 
```
nitpick: Add a link to the word 'diff' since new programmers may not know the meaning + reworded to be a bit smoother of a sentence (although that's a personal preference)

Author: Thanks for the suggestion, I like this idea!

>   For a :wikipedia:`diff </Diff>` of the changes and additional information, see the 

Author: ```suggestion
  For a :wikipedia:`diff <Diff>` of the changes and additional information, see the 
```
endpoints always suffix a slash, so not required here (actually breaks the link)

  use the `Realm Flutter and Dart Samples <https://github.com/realm/realm-dart-samples>`__ 
  and modify the sample project ``provider_shopper``.

.. note:: No Sync Support

>    The preview SDK does not support {+sync+}.

Reviewer: Instead of getting rid of this note altogether, could we replace the sentence in the note with "The alpha SDK does not support {+sync+} yet."? Since sync is not supported and the note would make this more visible to readers (even though I realize there is a bullet point for it, it might help to have a note as well)

Author: The bullet point is consistent with our explanation of "no sync support" in the Kotlin Multiplatform SDK, which the SDK team specifically requested because a warning style note was thought to be too much of a deterrent to potential users. (Imagine wanting to try out a product and seeing tons of warnings on the landing page -- you might change your mind, too!)
